index.js

```diff
@@ -16,3 +17,228 @@ mobileLinks.forEach((link) => {
+  id: '2',
+  img: './images/Snapshoot-Portfolio3.svg',
+  name: 'Facebook 360',
+  tech: ['Codekit', 'GitHub', 'JavaScript'],
+  description:
+    'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry`s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry`s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea',
+  languages: ['HTML', 'CSS', 'JavaScript'],
+  demo: 'abc',
+  source: 'def',
+};
+
+projects.push(projectThree);
+
+const projectFour = {
+  id: '3',
+  img: './images/Snapshoot-Portfolio4.svg',
+  name: 'Uber Navigation',
+  tech: ['Codekit', 'GitHub', 'JavaScript'],
+  description:
+    'Lorem Ipsum is simply dummy text of the printing and typesetting industry.',
+  languages: ['HTML', 'CSS', 'JavaScript'],
+  demo: 'abc',
+  source: 'def',
+};
+
+projects.push(projectFour);
+/* eslint-disable */
+function projectClick(event) 
+{
+  const clickedId = event.target.id;
+  header.classList.toggle('header-toggler');
+
+  for (let i = 0; i < projects.length; i + 1) {
+    if (projects[i].id === clickedId) {
+      const body = document.querySelector('body');
+      const popupSection = document.createElement('section');
+      popupSection.className = 'popupSection';
+      body.appendChild(popupSection);
+
+      const insideContainer = document.createElement('div');
+      insideContainer.className = 'insideDiv card';
+      popupSection.appendChild(insideContainer);
+
+      const headerImageContainer = document.createElement('div');
+      headerImageContainer.className = 'headerImageContainer';
+
+      const careersTitle = document.createElement('h2');
+      careersTitle.className = 'card-title';
+      careersTitle.innerHTML = projects[i].name;
+      headerImageContainer.appendChild(careersTitle);
+
+      const closeButton = document.createElement('div');
+      closeButton.classList = 'close-icon-popup closeButton';
+      closeButton.innerHTML = '<i class="fa-sharp fa-solid fa-xmark" onclick="closeWindow()"></i>';
+      headerImageContainer.append(closeButton);
+
+      insideContainer.appendChild(headerImageContainer);
+
+      const techList = document.createElement('div');
+      techList.className = 'list-inline p-1 d-flex';
+
+      for (let j = 0; j < projects[i].tech.length; j + 1) {
+        const list = document.createElement('p');
+        list.className = 'top-popup-para';
+        list.innerHTML = projects[i].tech[j];
+        const k = 0;
+        const counterContainer = document.createElement('div');
+        while (k < j) {
+          counterContainer.className = 'counter-container';
+          counterContainer.innerHTML = '<img src="./images/Counter.svg" alt="counter">';
+          k + 1;
+        }
```

Comment:
This loop has a little bug as it runs infinitely. This is because the value of k never changes. So, you might need to define k with the "let" keyword to allow it to be modified and the replace the line `k + 1` with `k = k + 1`

Also, the enclosing for loop has the same bug as outline in the previous comment.